static/beta/prod/navigation/ansible-navigation.json

```diff
@@ -106,6 +106,25 @@
                 }
             ]
         },
+        {
+            "title": "Ansible Lightspeed with IBM watsonx Code Assistant",
+            "expandable": true,
+            "routes": [
+                {
+                    "id": "seatsAdminUi",
```

Comment:
@wscalf can we please add some descriptions to these new routes? Otherwise when users try to search for these new links they will look odd (we will eventually be enforcing descriptions in these navigation files).

Reply:
Alright, descriptions added (which included updating stage beta, which also needed a description, so one more file now)